src/main/java/winston/AddDeadlineCommand.java

```diff
@@ -0,0 +1,26 @@
+package winston;
+
+/**
+ * Represents the deadline command from the parent abstract class Command.
+ */
+public class AddDeadlineCommand extends Command {
+    private final String description;
```

Comment:
`private final String description;`
Not really related to coding standard violations, but there might be a need to edit the task naming hence I feel that setting it as final may not be ideal.